packages/select/docs/FilterBarExample/DemographicValueSelect.tsx

```diff
@@ -74,8 +74,18 @@ export const DemographicValueSelect = ({
         <>
           <FilterMultiSelect.SearchInput />
           <FilterMultiSelect.ListBox>
-            {({ selectedItems, unselectedItems, disabledItems }) => (
+            {({
+              selectedItems,
+              unselectedItems,
+              disabledItems,
+              hasNoItems,
+            }) => (
               <>
+                {hasNoItems && (
+                  <FilterMultiSelect.NoResults>
+                    No results found.
```

Comment:
I think this would be nicer as a default message, so the consumer wouldn't need to add this all the time, but give them the option to override it with a custom one if the designer wants to provide more context to the items.

Reply:
Cass commented the same thing just below 👇 check out that thread

Reply:
Since it's not baked into the component, and Doug has raised some other points about why it can't seem to live there, we can't really do it sadly :')

Reply:
Sadness